app/controllers/solr_task_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SolrTaskController < ApplicationController
+  def index; end
+
+  def run_task
+    ENV["SOLR_CORE"] = "blacklight-core"
```

Comment:
I'm worried that this is very different from how the other services connect to Solr, but maybe that's ok if we're going to refactor it?